Simlar/Simlar-Info.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>NSCameraUsageDescription</key>
+	<string>Simlae uses your camera during video calls.</string>
```

Comment:
Simlar uses your camera during Simlar video calls so that the person you are calling can see you.